src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -6132,18 +6130,44 @@ def get_subscription_locations(cli_ctx):
     return [item.name for item in result]
 
 
-def list_flexconsumption_locations(cmd, zone_redundant=False):
+def list_flexconsumption_locations(cmd, zone_redundant=False, details=False, runtime=None):
+    client = web_client_factory(cmd.cli_ctx)
+
+    if runtime and not details:
+        raise ArgumentUsageError(
+            '--runtime is only valid with --details parameter. '
+            'Please try again without the --details parameter.')
+
+    regions = client.list_geo_regions(sku="FlexConsumption")
```

Comment:
Does this make the API call to 2023-01-01? I thought it wasn't and that was why we had to the raw API call below? Otherwise, I can't remember anymore why we were just doing the raw API call. 

```
geo_regions_api = 'subscriptions/{}/providers/Microsoft.Web/geoRegions?sku=FlexConsumption&api-version=2023-01-01'
request_url = cmd.cli_ctx.cloud.endpoints.resource_manager + geo_regions_api.format(sub_id)
flex_regions = send_raw_request(cmd.cli_ctx, "GET", request_url).json()['value']
```

Reply:
It is now making a call to 2023-01-01.